lgc/include/lgc/state/AbiMetadata.h

```diff
@@ -62,7 +68,9 @@ enum PipelineType : unsigned {
   NggTess,
   Mesh,
   TaskMesh,
+  Invalid
```

Comment:
I'm not sure I understand 100% what this is about yet, but I think this should probably be called `Partial` instead of `Invalid` for clarity because it occurs due to partial pipeline compilation.

Reply:
Remove.